src/code/z_skin_awb.c

```diff
@@ -1,13 +1,204 @@
 #include "global.h"
+#include "z64skin.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_skin_awb/func_80138410.s")
+void func_80138410(Skin* skin) {
```

Comment:
ClearSomething?

Reply:
this function doesnt exist in oot-- what context does it get used in?

Reply:
It is only used in `EnHorse_Init`
https://github.com/zeldaret/mm/pull/413/files#diff-1dac57939127037837e51b46d879fe422dfed7fb223d76d7b76fb4c6c4c12eb5R716

I'm not really sure why they decided to add it, it feels a bit unnecessary.

Reply:
I think something like init or setup or clear, then.

Reply:
We already have `Skin_Init`.
I guess I'll use `Skin_Setup`